.travis.yml

```diff
@@ -42,6 +43,10 @@ matrix:
   allow_failures:
     - php: hhvm
 
+install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
```

Comment:
> linebreak this

I was afraid of a make-like behavior where you loose the cwd on each line. Is this the case ?

> use long options.

Which ones ?

> use $() instead of backticks (they are deprecated)

Okay so this is a bash shell. Good to know.